module/VuFind/src/VuFind/RecordTab/AbstractContent.php

```diff
@@ -96,7 +96,7 @@ public function isActive()
     /**
      * Get content for ISBN.
      *
-     * @return array
+     * @return array|string
```

Comment:
I see that if we return an array, we escape the values, and if we return a string, we treat it as raw HTML. Thanks for correcting the return type. I think we should also probably document this behavior in the comment here, since it is a bit unintuitive.

Reply:
In fact this change here is a mistake. The change should be made in Content\AbstractBase class. RecordTab\AbstractContent is responsible for getting data from all providers (if more providers are set) and so returns always array (of arrays or of strings). Should be fixed now.